integration/caching_test.go

```diff
@@ -261,77 +261,4 @@ func testCaching(t *testing.T, context spec.G, it spec.S) {
 			})
 		})
 	})
-	context("a fixture is pushed twice", func() {
```

Comment:
Why are we dropping this test?

Reply:
It is identical to the one above it in every way.

Reply:
👀